src/components/App.js

```diff
@@ -0,0 +1,43 @@
+import { getList } from "../api";
+import ProductList from "./ProductList";
+import { useEffect, useState } from "react";
+
+const pageSize = 10;
+
+function App() {
+  const [orderBy, setOrderBy] = useState("recent");
+  const [items, setItems] = useState([]);
+  const [page, setpage] = useState(1);
+
+  const sortedItems = items?.sort((a, b) => b[orderBy] - a[orderBy]);
+
+  const handleNewestClick = () => setOrderBy("recent");
+  const handleBestClick = () => setOrderBy("favorite");
+
+  const handleLoad = async (options) => {
+    const { list } = await getList(options);
+
+    setItems(list);
+    setpage(options.page + 1);
+  };
```

Comment:
- 어떤 역할을 하는 지 함수이름으로 드러나면 좋겠네요
- option 중에 필요한 값만 받아도 될거 같네요

```suggestion
  const fetchProductList = async (options = {}) => {
    const { list } = await getList({ orderBy, page, pageSize: pageSize, ...options });

    setItems(list);
    setpage(options.page + 1);
  };
```